src/seedu/addressbook/data/person/Person.java

```diff
@@ -9,12 +9,16 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
+    
+    private static int nextSequenceNumber = 0;
```

Comment:
It would be more appropriate to let the `nextSequenceNumber` start from `1` instead of `0`